<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaneChangeEstimatorClass</class>
 <widget class="QMainWindow" name="LaneChangeEstimatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LaneChangeEstimator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Mode</string>
    </property>
    <item>
     <property name="text">
      <string>Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Line Extractor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trainer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tester</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>270</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Simulator by Hanwool WOO</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>METHOD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Changing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Keeping</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HMM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SVM</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lanechangeestimator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>handleTypeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>handleModeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox3</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>handleMethodChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>handleTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>handleModeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open</sender>
   <signal>triggered()</signal>
   <receiver>LaneChangeEstimatorClass</receiver>
   <slot>openMenuClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleModeChanged(int)</slot>
  <slot>handleModeSelected()</slot>
  <slot>openMenuClicked()</slot>
  <slot>handleTypeChanged(int)</slot>
  <slot>handleTypeSelected()</slot>
  <slot>handleMethodChanged(int)</slot>
 </slots>
</ui>
